extending.html

```diff
@@ -2224,15 +2224,16 @@ <h4 id="_creating_a_kar_file"><a class="anchor" href="#_creating_a_kar_file"></a
                     same feature twice.
                     Refer to Karaf forum posting at http://karaf.922171.n3.nabble.com/Duplicate-feature-repository-entry-using-archive-kar-to-build-deployable-applications-td3650850.html
                     --&gt;</span>
+                    <span class="tag">&lt;resourcesDir&gt;</span>${project.build.directory}/doesNotExist<span class="tag">&lt;/resourcesDir&gt;</span>
 
                     <span class="comment">&lt;!--
-                    Location of the features.xml file. If it references properties that need to be filtered, e.g., 2.9.2, it will need to be
+                    Location of the features.xml file. If it references properties that need to be filtered, e.g., ${project.version}, it will need to be
                     filtered by the maven-resources-plugin.
                     --&gt;</span>
-                    <span class="tag">&lt;featuresFile&gt;</span>{directory}/features.xml<span class="tag">&lt;/featuresFile&gt;</span>
+                    <span class="tag">&lt;featuresFile&gt;</span>${basedir}/target/classes/features.xml<span class="tag">&lt;/featuresFile&gt;</span>
 
-                    <span class="comment">&lt;!-- Name of the kar file (.kar extension added by default). If not specified, defaults to docs-2.9.2 --&gt;</span>
-                    <span class="tag">&lt;finalName&gt;</span>ddf-ifis-2.9.2<span class="tag">&lt;/finalName&gt;</span>
+                    <span class="comment">&lt;!-- Name of the kar file (.kar extension added by default). If not specified, defaults to ${project.build.finalName} --&gt;</span>
+                    <span class="tag">&lt;finalName&gt;</span>ddf-ifis-2.9.3<span class="tag">&lt;/finalName&gt;</span>
```

Comment:
I don't know. I assumed it was an example name. It can be updated in future versions to something meaningful.